src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/SuppressedIssuesFileSynchronizer.cs

```diff
@@ -82,13 +82,13 @@ private void SuppressedIssuesMonitor_SuppressionsUpdateRequested(object sender,
 
         public void UpdateFileStorage()
         {
-            threadHandling.Run(() =>
+            threadHandling.Run(async () =>
```

Comment:
This doesn't look right - we shouldn't be using the blocking `Run` here when we're calling something that is `awaitable`.
Or is this part of the "issues are fetched on the UI" problem that is going to be fixed separately?

Reply:
Yes this should be part of the UI thread issue.

Reply:
It might be better if this method were `async` too. However, that moves the problem to the call in `SuppressedIssuesMonitor_SuppressionsUpdateRequested`, which is a `void` event handler that needs to fire-and-forget the `async UpdateFileStorage` method on another thread.
There are few places in the code where we use the VS `.Forget()` method from `void` methods. Would that approach work?

Reply:
Ok.